flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/utils/RexNodeExtractorTest.scala

```diff
@@ -79,7 +78,7 @@ class RexNodeExtractorTest extends RexNodeTestBase {
     val usedFields = RexNodeExtractor.extractRefInputFields(rexProgram)
     val usedNestedFields = RexNodeExtractor.extractRefNestedInputFields(rexProgram, usedFields)
 
-    val expected = Array(Array("amount"), Array("*"))
+    val expected = Array(Array(util.Arrays.asList("amount")), Array(util.Arrays.asList("*")))
```

Comment:
nit: please delete a unused method `assertPlannerExpressionArrayEquals`

Reply:
I has rm the unused method.